### ISSUE TITLE
can you implement time and date ? 

### ISSUE THREAD

**Issue by protowax** [programmerraja/IpLogger#1]
can you implement time and date in the script 
thanks

**Comment by programmerraja** [comment]
Do you want to know the time that user check the link?

**Comment by protowax** [comment]
Yes plz and if there's a contact info to contact you directly about some ideas thanks

**Comment by programmerraja** [comment]
Are you available in telegram?

https://t.me/programmerraja
